src/components/calculator.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Btn = (props) => {
+  const { btnName } = props;
+  return (
+    <div className="btn-div">
+      <button type="button" className="btn">{btnName}</button>
+    </div>
+  );
+};
+
+Btn.propTypes = {
+  btnName: PropTypes.string.isRequired,
+};
+
+const buttonArr = ['AC', '+/-', '%', '÷', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];
+
+const Calculator = () => (
+  <div className="calc">
+    <input type="text" placeholder={0} className="calc-input" />
+    <div className="btn-contain">
+      {buttonArr.map((btn) => (
+        <Btn key={btn} btnName={btn} />
+      ))}
+    </div>
+  </div>
+);
+
+export default Calculator;
```

Comment:
- [ ] Nice to use functional components but in this milestone, you should use class based components. You will refactor it in next milestones[Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).)